Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Create builder image
+FROM node:16-alpine AS builder
+WORKDIR /build
+
+# Install dependencies
+COPY package.json yarn.lock ./
+RUN yarn --frozen-lockfile
+
+# Copy remaining required files and create production build
+COPY src ./src/
+COPY webpack.config.js ./
+COPY CHANGELOG.md ./
+COPY LICENSE ./
+COPY README.md ./
+RUN yarn run build
+
+# Copy production build into an nginx image
+FROM nginx
```

Comment:
Same here for the `docker.io/library/` prefix.
Can you also use an explicit tag instead of using the latest one, to avoid any breaking changes in the future?